src/main/java/gyro/aws/neptune/NeptuneClusterParameterGroupResource.java

```diff
@@ -0,0 +1,381 @@
+        this.enableAuditLog = enableAuditLog;
+    }
+
+    /**
+     * Accept SSL/TLS connections only. Valid values are ``0`` or ``1``.
+     * Defaults to 1.
+     *
+     * @subresource gyro.aws.neptune.NeptuneParameter
+     */
+    @Updatable
+    public NeptuneParameter getEnforceSsl() {
+        if (enforceSsl != null) {
+            enforceSsl.setName("neptune_enforce_ssl");
+            enforceSsl.setApplyMethod("pending-reboot");
+        }
+        return enforceSsl;
+    }
+
+    public void setEnforceSsl(NeptuneParameter enforceSsl) {
+        this.enforceSsl = enforceSsl;
+    }
+
+    /**
+     * Toggle Neptune engine experimental features.
+     * Value is a comma-separated list including ``(feature name)=enabled`` or ``(feature name)=disabled``.
+     * Valid feature names include ``ObjectIndex``, ``Streams``, and ``ReadWriteConflictDetection``.
+     *
+     * @subresource gyro.aws.neptune.NeptuneParameter
+     */
+    @Updatable
+    public NeptuneParameter getLabMode() {
+        if (labMode != null) {
+            labMode.setName("neptune_lab_mode");
+            labMode.setApplyMethod("pending-reboot");
+        }
+        return labMode;
+    }
+
+    public void setLabMode(NeptuneParameter labMode) {
+        this.labMode = labMode;
+    }
+
+    /**
+     * Graph query timeout (ms). Valid values range from ``10`` to ``2147483647``.
+     * Defaults to ``120000``.
+     *
+     * @subresource gyro.aws.neptune.NeptuneParameter
+     */
+    @Updatable
+    public NeptuneParameter getQueryTimeout() {
+        if (queryTimeout != null) {
+            queryTimeout.setName("neptune_query_timeout");
+            queryTimeout.setApplyMethod("pending-reboot");
+        }
+        return queryTimeout;
+    }
+
+    public void setQueryTimeout(NeptuneParameter queryTimeout) {
+        this.queryTimeout = queryTimeout;
+    }
+
+    @Override
+    public void copyFrom(DBClusterParameterGroup model) {
+        setFamily(model.dbParameterGroupFamily());
+        setName(model.dbClusterParameterGroupName());
+        setDescription(model.description());
+        setArn(model.dbClusterParameterGroupArn());
+        loadParameters();
+    }
+
+    @Override
+    protected boolean doRefresh() {
+        DBClusterParameterGroup group = getDbClusterParameterGroup();
+        if (group == null) {
+            return false;
+        }
+
+        copyFrom(group);
+        return true;
+    }
+
+    @Override
+    protected void doCreate(GyroUI ui, State state) {
+        NeptuneClient client = createClient(NeptuneClient.class);
+        CreateDbClusterParameterGroupResponse createResponse = client.createDBClusterParameterGroup(
+            r -> r.dbClusterParameterGroupName(getName())
+                .dbParameterGroupFamily(getFamily())
+                .description(getDescription())
+        );
+
+        setArn(createResponse.dbClusterParameterGroup().dbClusterParameterGroupArn());
+        saveParameters();
```

Comment:
If the create fails after creating the param group but before saving the parameters the corresponding gyro up should pick it up from where it started.
```suggestion
        state.save();
        
        saveParameters();
```